src/components/ScrollButton.module.css

```diff
@@ -0,0 +1,27 @@
+.scrollButton {
+	position: fixed;
+	display: flex;
+	bottom: 40px;
+	right: 40px;
+	background: var(--colors-background-bold);
+	border-radius: 4px;
+	border-radius: var(--border-radius-large);
+	border: none;
+	color: var(--color-background);
+	cursor: pointer;
+	font-weight: bold;
+	justify-content: center;
+	padding: 0.5rem 0.75rem;
+	text-align: center;
+	text-decoration: none;
+	text-shadow: 0 0 5px var(--color-shadow);
+}
+
+@media screen and (width <= 870px) {
+	.scrollButton {
+		position: fixed;
+		bottom: 40px;
+		right: 40%;
+		left: 40%;
+	}
+}
```

Comment:
LOL. Not a blocker if you don't want to spend a bunch of time on it.


https://github.com/JoshuaKGoldberg/tidelift-me-up-site/assets/3335181/d209768c-f866-4e3f-a412-81ec0797352a